src/CommandingOfficers/DocLight.java

```diff
@@ -0,0 +1,104 @@
+package CommandingOfficers;
+
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEventQueue;
+import UI.UIUtils;
+import Units.*;
+
+public class DocLight extends RuinedCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Doctor Light", UIUtils.MISC);
+      infoPages.add(new InfoPage(
+            "Even here, it is not safe. Even this grave has been defaced.\n"
+          + "Someone has written on this stone: 'Hope rides alone'\n"));
+      infoPages.add(new InfoPage(
+          "Base Zone: 0\n" +
+          "Can deploy your COU only twice, with different stats each time (Protoman is first)\n" +
+          "Protoman Boost: +"+BLUES_POWER+"/"+BLUES_POWER+" stats and +0/"+BLUES_SHIELD+" vs indirect attack ("+(BLUES_POWER+130)+"/"+(BLUES_POWER+BLUES_SHIELD+130)+" vs indirects)\n" +
+          "Megaman Boost: +"+ROCK_POWER+"/"+ROCK_POWER+" stats ("+(ROCK_POWER+130)+"/"+(ROCK_POWER+130)+" total)\n"));
```

Comment:
Is the Megaman boost on top of the Protoman boost? The wording is kinda confusing: +50, but 180 total.

Reply:
Generic zone bonus: +10/10
Veterancy bonus (COUs are auto-vet, even if veterancy is disabled): +20/20
Megaman boost (which matches Tabitha and Caulder): +50/50
COU Statline: 180/180
Tabi/Caulder zone statline: 160/160

Reply:
OK, so this Megaman has higher stats than the Tabi/Caulder COU (180 vs 169), but doesn't affect any other units? The phrasing is still strange for anyone not intimately familiar with the DoR mechanics. Again I'd lean towards being a bit more verbose for clarity.

Reply:
No, he has the same stats.
+10/10 generic
+50/50 specific (universal boost)
This accounts for the 160/160 Tabi/Caulder zone stats
Max veterancy adds an extra +20/20, and always applies to the COU so:
180/180
This statline applies to Megaman, Caulder, and Tabitha.
Caulder and Tabitha can have zone units that aren't veterans, though, which is why I added that line.

I mean, that's why I wrote that big ol' blurb on DoR mechanics that gets appended to the descriptions.
It's just a lot to unpack if you're going to write out everything explicitly.

Reply:
Right. So I'd probably either keep it DRY (only mention DoR mechanics in the DoR section, and CO stat boosts in the CO stat boost section, and let the user add it all up), or specifically state how a +50 magically turns into a 180 total ("... with the DoR zone and veterancy bonuses..."). Currently you are just giving half of the story, which is arguably more confusing than skipping it altogether.

But you probably know your target audience better than I do, so if this is standard parlance and perfectly clear to people who are not me, I'll try not to nitpick too much. I guess up to you if it's worth clarifying, but other than that I guess it's ready to go (unless you want to redesign the CO's mechanics first...).